python/GafferDispatchTest/StatsApplicationTest.py

```diff
@@ -0,0 +1,76 @@
+##########################################################################
+#
+#  Copyright (c) 2016, Image Engine Design Inc. All rights reserved.
```

Comment:
Hot Tub Time Machine :duck:

Reply:
Integrity of space time continuum restored in 654d5ba.